arch/arm/boot/dts/bcm2711-rpi-cm4-fr201.dts

```diff
@@ -2,11 +2,20 @@
 
 /* Nodes for OnLogic FR201 device */
 
-/* Power LED */
-&leds {
-	led-pwr {
-		default-state = "off";
-		linux,default-trigger = "none";
+/* LEDs */
+/ {
+	leds {
+		act_led: led-act {
+			default-state = "off";
+			linux,default-trigger = "none";
+			gpios = <&gpio 42 GPIO_ACTIVE_HIGH>;
+		};
+
+		pwr_led: led-pwr {
+			default-state = "off";
```

Comment:
@rene  I think the commit message is a bit misleading. "PWR should be ON by default"

Reply:
Thanks @rucoder... I was referring to the default trigger mode. It's fixed, I think it's more clear now...

Reply:
@rene ah, yes, looks better now! (i would fix PR description as well)

Reply:
@rene I've fixed it for you. waiting for builds to finish before merging